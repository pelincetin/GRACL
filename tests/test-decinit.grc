int glob = 123;
String me = "Are you";
int main()
{
  int a = 42;
  String you = "me?";
  printi(glob);
  print(me);
  print(you);
  printi(a);
}