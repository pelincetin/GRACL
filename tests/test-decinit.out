123
Are you
me?
42
47
2.718000